tests/nightly/nightly-autorun.sh

```diff
@@ -16,8 +16,9 @@
 
 # Get the latest code from vmware/vic repo
 cd ~/go/src/github.com/vmware/vic
-git fetch
-git reset --hard origin/master
+sudo git clean -fd
```

Comment:
@mhagen-vmware, also, do we need to change where all of the nighty_$DATE_.txt files are stored?  That way we can keep logs for troubleshooting.